lms/templates/ux/reference/course-skeleton.html

```diff
@@ -0,0 +1,71 @@
+## Override the default styles_version to the Pattern Library version (version 2)
+<%! main_css = "style-main-v2" %>
+
+<%page expression_filter="h"/>
+<%inherit file="/main.html" />
+
+<%block name="pagetitle">Course Skeleton</%block>
+
+<%block name="bodyclass">pattern-library</%block>
+
+<%block name="content">
+<nav class="wrapper-course-material">
+    <div class="course-material">
+        <ol class="tabs course-tabs">
+            <li class="tab">
+                <a href="index.html" class="">
```

Comment:
It might be best to include a class then target that in the Sass. I think we're trying to move away from targeting elements.